roxctl/central/db/restore/v2_test.go

```diff
@@ -0,0 +1,36 @@
+package restore
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCentralDBRestore_Validate(t *testing.T) {
+	cmd := &centralDbRestoreCommand{}
+
+	// 1. If file is unset, expect an InvalidArgs error.
```

Comment:
nit: How about wrapping them in subtest with this comment as a name? We can then run this tests in parallel if they do not share cmd